.circleci/config.yml

```diff
@@ -28,6 +31,18 @@ jobs:
             git tag -a v$(cat version.txt) -m "Release version $(cat version.txt)"
             git push --tags origin
             git push origin master
+  build_client:
+    executor: client-builder
+    working_directory: ~/ara/client
+    steps:
+      - checkout:
+          path: ~/ara
```

Comment:
This step seems weird to me. If I understand it : 
- We're checking out all the project inside the path `~/ara` but we're working only in `~/ara/client` 
amiright ?

If so, maybe we should remove the checkout step from this job and the `build_app` one to put it in its own job. Doing so will allow use to only checkout once when we'll use them both (like in `build-pr` and `build-master`. What's your opinion on this ?

Reply:
I had some trouble reaching the client directory: even though setting working_directory as ~/ara/client, without the checkout, I wasn't able to access it.
This helped resolve the issue -> https://stackoverflow.com/questions/50570221/circleci-2-0-working-with-subdirectory

Reply:
Ok for me, but could you has the reason and the link in comment in the YML file please ?

Reply:
done